addons/binding/org.openhab.binding.freebox/ESH-INF/thing/thing-types.xml

```diff
@@ -34,6 +34,15 @@
 			<channel id="bytes_down" typeId="bytes_down" />
 		</channels>
 
+		<properties>
+			<property name="vendor">Freebox SAS</property>
+			<property name="serialNumber"></property>
```

Comment:
I do not like this - properties should not have an empty string as value. Either they are set or not.